### PR TITLE
feat(samples): add pagination samples

### PR DESCRIPTION
**Pull request by FrodoTheTrue**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,7 +109,6 @@
     "predocs-test": "npm run docs",
     "prepare": "npm run compile-protos && npm run compile",
     "prelint": "cd samples; npm link ../; npm install",
-    "clean": "gts clean",
```
Comment (fhinkel) [inline package.json:112]: why did you remove this script?

File: samples/listImages.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 Google LLC
```
Comment (fhinkel) [inline samples/listImages.js:1]: Can you commit the tests for the two new sample files please.

**Review by fhinkel — CHANGES_REQUESTED** [review]
Please commit the test files as well.

File: package.json (hunk 1 above)
Comment (FrodoTheTrue) [reply package.json:112]: My git system automatically deleted it.
So, I see that "clean" command using twice in package.json for now.

**Review by fhinkel — APPROVED** [review]

**Comment by fhinkel** [comment]
Added the do-not-merge label, because the tests are green without showing the github action tests after clicking "rerun".

**Comment by fhinkel** [comment]
FYI, the tests are not running because you're merging to non-main branch.

**Review by bcoe — APPROVED** [review]
